sound-equipment-fault-detection/vled/README.md

```diff
@@ -0,0 +1,52 @@
+# vled 🎇🎇🎇
```

Comment:
It seems that you have implemented a vled mapper based on mapper-framework, right? Are you willing to submit the vled mapper to the [mappers-go repo](https://github.com/kubeedge/mappers-go) as well for reference by other developers?

Reply:
Yes, I am happy to contribute to kubeedge!